doc/man1/openssl-cmp.pod.in

```diff
@@ -624,6 +625,15 @@ For Key Update Request (KUR) messages this is also used as
 the certificate to be updated if the B<-oldcert> option is not given.
 If the file includes further certs, they are appended to the untrusted certs.
 
+=item B<-own_trusted> I<filenames>
+
+If this list of certificates are provided they are used to verify the
+chain building for the own CMP signer certificate.
```

Comment:
Its a singular list containing plural certificates. Since list is singular I think this should be  "...list of certificate *is* provided".

Should it be "for our own CMP signer certificate"?

Reply:
Good point the grammar should be improved. I've re-phrased the sentence to:
```
If this list of certificates is provided then the chain built for the own CMP
signer certificate is verified using the given certificates as trust anchors.
```